plugins/woocommerce-admin/client/task-lists/fills/tax/woocommerce-tax/plugins.tsx

```diff
@@ -99,16 +89,21 @@ export const Plugins: React.FC< SetupStepProps > = ( {
 				skipText={ __( 'Set up manually', 'woocommerce' ) }
 				onAbort={ () => onDisable() }
 				abortText={ __( "I don't charge sales tax", 'woocommerce' ) }
+				pluginSlugs={ pluginsToActivate }
 			/>
 			{ ! tosAccepted && (
 				<Text
 					variant="caption"
 					className="woocommerce-task__caption"
 					size="12"
 					lineHeight="16px"
+					style={ { display: 'block' } }
```

Comment:
This is a bugfix that I'm sneaking in here.

Previously:

![Screenshot 2023-09-29 at 13-25-36 Home ‹ WooCommerce ‹ Fulfill — WooCommerce](https://github.com/woocommerce/woocommerce/assets/1759681/d782e7cb-6088-40db-b45c-3532d91c86a1)

With this fix:

![Screenshot 2023-09-29 at 13-22-46 Home ‹ WooCommerce ‹ Fulfill — WooCommerce](https://github.com/woocommerce/woocommerce/assets/1759681/8babf732-9f11-4572-8976-8f37a156d0c3)